src/main/resources/mustache-templates/layout/footer.html

```diff
@@ -1,19 +1,37 @@
+
+<footer class="page-footer font-small mdb-color lighten-3 pt-4">
 	<div class="container">
-		<footer>
-			<p class="pull-right"><small>HPO on FHIR</small></p>
-		</footer>
+		<div class="row">
+			<div class="col-sm-6 footer-copyright py-3">
+				© 2018: <a href="https://github.com/OCTRI/poc-hpo-on-fhir"> HPO
```

Comment:
The current year could be added as a context variable using a class with the ControllerAdvice annotation.

Reply:
Nice! I hadn't used this before. Your code in CDCA was a useful guide.